pages/index.js

```diff
@@ -41,9 +44,67 @@ class Index extends Component {
                     integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
                     crossorigin="anonymous"
                 />
-                <NavbarBootstrap
-                    isAuthenticated={isAuthenticated}
-                    name={name}
+                <link
+                    rel="stylesheet"
+                    href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"
+                    integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp"
+                    crossorigin="anonymous"
```

Comment:
I found console errors relating this. It asks to change this to `crossOrigin`. Please have a look 😊

Reply:
good catch, thanks